src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -82,6 +82,12 @@ private function decideAffirmative(TokenInterface $token, array $attributes, $ob
         $deny = 0;
         foreach ($this->voters as $voter) {
             $result = $voter->vote($token, $object, $attributes);
+            if (false === \is_int($result)) {
+                @trigger_error(
+                    sprintf('Return value of "%s::vote()" should always be of the type int since Symfony 5.0, %s returned.', \get_class($voter), \gettype($result)),
+                    \E_USER_DEPRECATED
+                );
```

Comment:
```suggestion
                trigger_deprecation('symfony/security-core', '5.3', 'Returning a value of type "%s" in "%s::vote()" is deprecated, return an int instead.', get_debug_type($result), \get_class($voter));
```

Reply:
now I am lost. 5.3 is supposed to be for PHP 8.0 only??

Reply:
We have a Polyfill for `get_debug_type()`. You can safely use that method on PHP 7 as well.

Reply:
ah perfect. I allowed myself to write to you on slack

Reply:
FYI: https://github.com/symfony/polyfill-php80